src/core/thread/osthread.d

```diff
@@ -1579,8 +1648,27 @@ private:
         sm_this = t;
     }
 
+package(core.thread):
```

Comment:
One of the reasons for wanting to do this refactoring in the first place was to have this be pubic so that Weka can use it in their own fiber code. Please remove this line.

Reply:
and move it outside of `class Thread`